platform-service/src/unstract/platform_service/helper/cost_calculation.py

```diff
@@ -49,24 +76,47 @@ def calculate_cost(
 
     def _get_model_token_data(self) -> Optional[dict[str, Any]]:
         try:
-            # File does not exist, fetch JSON data from API
-            if not os.path.exists(self.file_path):
-                return self._fetch_and_save_json()
+            if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
```

Comment:
@gaya3-zipstack to prevent this function from growing in size and to maintain readibility please split as 2 functions

Reply:
@gaya3-zipstack I think its okay to not add a new function if its just a few lines of changes, however a large number of lines added should be in a separate function. We don't have a fixed coding standard around the number of lines allowed in a function yet which causes this ambiguity. My only worry is to not end up in a (nightmare-ish) scenario similar to prompt-service's `main.py` :sweat_smile: 
Will let you guys take a call on this wrt these temporary changes